### PR TITLE
remove kathmandu references

### LINKED ISSUE #2131: Remove Kathmandu/Proto14 conditions in taquito-local-forging
Remove the Kathmandu/Proto14 condition that was required for the transition from Kathmandu to Lima

integration-tests/data/allTestsCases.ts
integration-tests/data/code_with_ticket_proto14.ts
integration-tests/local-forging.spec.ts
integration-tests/wallet-edo-deploy-having-ticket.spec.ts
packages/taquito-local-forging/src/proto14-kathmandu/
packages/taquito-local-forging/src/taquito-local-forging.ts
packages/taquito-local-forging/test/michelson/encoder.spec.ts
packages/taquito-local-forging/test/taquito-local-forging.spec.ts

### PR DESCRIPTION
**Pull request by dsawali**
closes #2131 

Thank you for your contribution to Taquito.

Before submitting this PR, please make sure:

- [x] Your code builds cleanly without any errors or warnings
- [x] You have run the linter against the changes
- [x] You have added unit tests (if relevant/appropriate)
- [ ] You have added integration tests (if relevant/appropriate)
- [ ] All public methods or types have TypeDoc coverage with a complete description, and ideally an @example
- [ ] You have added or updated corresponding documentation
- [ ] If relevant, you have written a first draft summary describing the change for inclusion in Release Notes. 

In this PR, please also make sure: 

- [ ] You have linked this PR to the issue by putting `closes #TICKETNUMBER` in the description box (when applicable)
- [ ] You have added a concise description on your changes
## Release Note Draft Snippet

__If relevant, please write a summary of your change that will be suitable for
inclusion in the Release Notes for the next Taquito release.__

### CODE REVIEW

File: integration-tests/data/allTestsCases.ts [hunk 1]
```diff
@@ -807,7 +796,7 @@ export const commonCases: TestCase[] = [
       ],
     },
   },
-  ...extractOp(0, 151, opMappingProto14).map((op): TestCase => {
+  ...extractOp(0, 154, opMapping).map((op): TestCase => {
```
Comment (zainen) [inline integration-tests/data/allTestsCases.ts:799]: just want to double check is 154 the length of the opMapping?
would it be better to keep this as opMapping.length - 1 in that case?

Comment (dsawali) [reply integration-tests/data/allTestsCases.ts:799]: opMapping is actually more than that, this is just the cutoff of what we tested in Lima

Comment (zainen) [reply integration-tests/data/allTestsCases.ts:799]: ahh ok! makes sense to test the ones specific for the protocol!

**Review by zainen — APPROVED** [review]
LGTM! nice and clean now :D

**Review by hui-an-yang — COMMENTED** [review]
lgtm🎉

**Review by hui-an-yang — APPROVED** [review]

**Pull request merged**